src/Explorer.js

```diff
@@ -495,11 +502,16 @@ class ArgView extends React.PureComponent<ArgViewProps, ArgViewState> {
         setArgValue={this._setArgValue}
         getDefaultScalarArgValue={this.props.getDefaultScalarArgValue}
         makeDefaultArg={this.props.makeDefaultArg}
+        onRunOperation={event => this.props.onRunOperation()}
       />
     );
   }
 }
 
+const isRunShortcut = event => {
```

Comment:
Function is more efficient and fewer characters

```
function isRunShortcut(event) {
  return event.metaKey && event.keyCode === 13;
}
```

Could you also add a comment about what keyCode 13 is.

Reply:
I'm dubious this is a meaningful improvement, and hoisting is pretty annoying. Still, changed it to:

```
function isRunShortcut(event) {
  return event.metaKey && event.key === 'Enter';
}
```